src/_pytest/timing.py

```diff
@@ -7,6 +7,47 @@
 """
 from time import perf_counter
 from time import sleep
-from time import time
+from time import time as Time
```

Comment:
those renames add a confusion factor

Reply:
How would you suggest me to approach this ?

Reply:
`from time import time as _time` ?